modern/src/common/util/converter.js

```diff
@@ -10,6 +10,18 @@ const speedConverter = (unit) => {
   }
 };
 
+const speedConverterKph = (unit) => {
```

Comment:
I don't think we need a whole new conversion function. You should be able to just do `value / speedConverter('kmh').